analysis/analysis-api/testData/symbols/singleSymbolByPsi/typeAnnotationsOnFunctionalTypeWithTypeAlias.pretty.txt

```diff
@@ -0,0 +1 @@
+fun foo(): @Anno3 (() -> kotlin.Unit)
```

Comment:
Not necessary to fix this right now in this PR, but it'd be better if AA FIR's pretty printer dumps all type annotations in a chain of type aliases.

Reply:
Done.

Reply:
Aha, so this is actually done by updating `annotationsList`? :)